app/src/main/java/fr/free/nrw/commons/achievements/AchievementsActivity.java

```diff
@@ -316,8 +316,8 @@ private void setUploadProgress(int uploadCount){
 
     private void setZeroAchievements() {
         AlertDialog.Builder builder=new AlertDialog.Builder(this)
-                .setMessage("You haven't made any contributions yet")
-                .setPositiveButton("Ok", (dialog, which) -> {
+                .setMessage(getString(R.string.achiev_activity_set_zero_achiev))
```

Comment:
I would suggest spelling the full word `achievements`, even if that means a longer variable name.